packages/liveblocks-client/src/room.ts

```diff
@@ -553,8 +553,6 @@ export function makeStateMachine(
         return parent._attachChild(op, isLocal);
       }
     }
-
-    return { modified: false };
```

Comment:
This removes a line of unreachable code. The added benefit here is that if we extend the `OpType` enum, TypeScript will force us to get back to this function and deal with it. In other words, not exhaustively handling these cases will be a TypeScript error.